README.md

```diff
@@ -4,10 +4,11 @@ Dark theme created by <a href="https://rocketseat.com.br?ref=omnitheme">Rocketse
 
 ## Themes
 
-Omni currently supports 16 applications. You can find them below:
+Omni currently supports 17 applications. You can find them below:
 
 ### Browsers
 
+- [Vivaldi](https://github.com/getomni/Vivaldi)
```

Comment:
Hey, thanks for opening the PR!

Could you place Vivaldi on the end of this browser list? We are ordering the themes alphabetically